### PR TITLE
add gating question on sale of assets (stocks, bonds, real estate)

### PR DESCRIPTION
**Pull request by loumoore**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ en:
       sessions_controller:
         new:
           invalid_email_or_password: Incorrect email or password. After 5 login attempts, accounts are locked for 30 minutes.
-  email:
-    user_message:
-      subject: "Update from GetYourRefund"
```
Comment (bengolder) [inline config/locales/en.yml:18]: Why did this get deleted? We'd have thought that any unused keys would be caught by the i18n specs already

File: app/controllers/questions/asset_sale_gate_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Questions
+  class AssetSaleGateController < TicketedQuestionsController
```
Comment (bengolder) [inline app/controllers/questions/asset_sale_gate_controller.rb:2]: Might be more consistent to just name this `AssetSaleController` or `SoldAssetsController` to be consistent with the rest of the controllers. We don't have any other explicit `Gate` controllers.

**Review by bengolder — APPROVED** [review]
Approved! with one small suggested non-blocking improvement.

File: app/controllers/questions/asset_sale_gate_controller.rb (hunk 2 above)
Comment (loumoore) [reply app/controllers/questions/asset_sale_gate_controller.rb:2]: would love thoughts on naming convention as there will be more gate controllers, but still not sure if it'll be clearer to folks if they are explicitly named "*Gate" or if I try to come up with a descriptive name for each. Designs [here]([https://www.figma.com/file/KhjUvgH13LaUjOHR8sTuRm/Get-Your-Refund?node-id=3503%3A0])

Comment (loumoore) [reply app/controllers/questions/asset_sale_gate_controller.rb]: went ahead and did this, refactored all the things (controller + form) from AssetSalesGate to SoldAssets

**Pull request merged**